clients/src/main/java/org/apache/kafka/common/security/oauthbearer/secured/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * This package is deprecated.
+ */
+package org.apache.kafka.common.security.oauthbearer.secured;
```

Comment:
You could add the `@Deprecated` annotation and the `@deprecated` Javadoc tag.

Reply:
I added the `@deprecated` Javadoc tag, but needed to leave the original `This package is deprecated.` otherwise the package listing was empty.

I also tried adding `@Deprecated` to the package name, and received a build error.
Let me know if the current state is still not acceptable.